Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3460,12 +3460,12 @@ IsFirstPartyWebsiteDataRemovalReproTestingEnabled:
       default: false
 
 # FIXME: The 'Is' prefix is inconsistent with most other preferences and should be removed.
-IsLoggedInAPIEnabled:
+LoginStatusAPIEnabled:
```

Comment:
It looks like this file is sorted by name. Since this was renamed we probably need to move it to keep alphabetical order.

Reply:
Which file?

Reply:
In this file (UnifiedWebPreferences.yaml). LoginStatusAPIEnabled likely needs to be moved below LockdownFontParserEnabled.